gov/agendas/tracker.go

```diff
@@ -467,6 +467,10 @@ func (tracker *VoteTracker) newVoteSummary() *VoteSummary {
 // never be modified by VoteTracker, so can be used read-only by any number
 // of threads.
 func (tracker *VoteTracker) Summary() *VoteSummary {
+	// lite mode hack
+	if tracker == nil {
+		return nil
```

Comment:
Resolving it this way since the proper fix is not needed at all on `master`/5.0 and lite mode is essentially never used.